SimTKcommon/include/SimTKcommon/internal/Array.h

```diff
@@ -3320,7 +3320,7 @@ std::istream& readArrayFromStreamHelper
 
     // Now see if the sequence is bare or surrounded by (), [], or {}.
     bool lookForCloser = true;
-    char openBracket, closeBracket;
+    char openBracket, closeBracket{};
```

Comment:
For better RAII, move this line down to where openBracket is initialized. I think it could read:
```c++
char openBracket{ch}, closeBracket{};
```